### PR TITLE
Added resources/sensors api call to list all sensors

### PR DESCRIPTION
**Pull request by zombah**
Simple api call to get full list of all sensors

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: html/includes/api_functions.inc.php [hunk 1]
```diff
@@ -2027,6 +2027,42 @@ function get_link()
...
+        $sql_params = array($device_id);
+    }
+    if (!LegacyAuth::user()->hasGlobalRead()) {
+        $sql .= " AND `sensors`.`device_id` IN (SELECT device_id FROM devices_perms WHERE user_id = ?)";
+        $sql_params[] = LegacyAuth::id();
+    }
+
+    $sensors       = array();
+    foreach (dbFetchRows("SELECT `sensors`.* FROM `sensors` LEFT JOIN `devices` ON `sensors`.`device_id` = `devices`.`device_id` WHERE `sensors`.`sensor_id` IS NOT NULL $sql", $sql_params) as $sensor) {
+        $host_id = get_vm_parent_id($device);
```
Comment (murrant) [inline html/includes/api_functions.inc.php]: $device is not defined yet...

Comment (zombah) [reply html/includes/api_functions.inc.php]: Yes, missed that. Now i see that i wrongly duplicated this construct, it looks devices table specific with it's ip and parent_id stuff. Maybe to just use check_is_read for such list functions?

File: html/includes/api_functions.inc.php [hunk 2]
```diff
@@ -2029,31 +2029,12 @@ function get_link()
...
-    $sensors       = array();
-    foreach (dbFetchRows("SELECT `sensors`.* FROM `sensors` LEFT JOIN `devices` ON `sensors`.`device_id` = `devices`.`device_id` WHERE `sensors`.`sensor_id` IS NOT NULL $sql", $sql_params) as $sensor) {
-        $host_id = get_vm_parent_id($device);
-        $device['ip'] = inet6_ntop($device['ip']);
-        if (is_numeric($host_id)) {
-            $device['parent_id'] = $host_id;
-        }
-        $sensors[] = $sensor;
-    }
+    $sensors       = dbFetchRows("SELECT `sensors`.* FROM `sensors` WHERE `sensors`.`sensor_id` IS NOT NULL);
```
Comment (murrant) [inline html/includes/api_functions.inc.php]: sensor_id cannot be null, fyi

Comment (murrant) [reply html/includes/api_functions.inc.php]: You could also use Eloquent here:

```php
$sensors = \App\Models\Sensor::hasAccess(Auth::user())->get();
$total_sensors = $sensors->count();
```

Comment (zombah) [reply html/includes/api_functions.inc.php]: @murrant MySQL schema says that NULL is default for sensor_id:
```
MariaDB [librenms]> describe sensors;
+---------------------------+------------------+------+-----+-------------------+-----------------------------+
| Field                     | Type             | Null | Key | Default           | Extra                       |
+---------------------------+------------------+------+-----+-------------------+-----------------------------+
| sensor_id                 | int(10) unsigned | NO   | PRI | NULL              | auto_increment              |
```
i think it's good to have such sanity check same way all other api calls have
or better not?

Comment (zombah) [reply html/includes/api_functions.inc.php]: @murrant Count is not working for some reason with Eloquent

Comment (murrant) [reply html/includes/api_functions.inc.php]: @zombah it is auto_increment and also not nullable.

Count works for me:
```
php artisan tinker
Psy Shell v0.9.9 (PHP 7.3.3 — cli) by Justin Hileman
>>> $sensors = \App\Models\Sensor::hasAccess(User::first())->get();
>>> $sensors->count();
=> 13
```

**Pull request merged**